src/cloud/docker/docker-quick-reference.md

```diff
@@ -49,13 +49,13 @@ See [Service versions] for a list of the options to configure the software servi
 
 ## bin/magento-docker
 
-Run `bin/magento-docker` commands using the following format:
+The `bin/magento-docker` commands simplify running docker-compose tasks. For example, instead of running a separate docker-compose command for the build, deploy, and post-deploy steps, you can _redeploy_ Magento in a Docker environment using the following command:
 
 ```bash
-./bin/magento-docker <command>
+./bin/magento-docker ece-redeploy
 ```
 
-For example, to connect to the bash shell:
+The following example shows the `./bin/magento-docker` command output when connecting to the bash shell:
```

Comment:
You say the following is the output, but the command is inside the output code block.